db/migrations/01_create_chitter_table.sql

```diff
@@ -0,0 +1,3 @@
+CREATE TABLE chitter ( id SERIAL PRIMARY KEY,  Peeps VARCHAR(140), 
+Time_Stamp TIMESTAMP WITHOUT TIME ZONE NOT NULL 
+DEFAULT (current_timestamp AT TIME ZONE 'UTC'));
```

Comment:
nice one, was thinking about doing similar thing (getting time directly from database not app but didnt spent much time trying to figure out how to do it :P)